data/human/free worlds 4 epilogue.txt

```diff
@@ -144,6 +144,51 @@ mission "FW Epilogue: Freya"
 				`	"You sound frustrated."`
 				`	"You know, the war is over, you can relax if you want to."`
 			`	"I know, I know," she says. "I complain, but really there's nowhere else I would rather be. Even if everything we're discovering here is nothing more than what they intended for us to discover, it's still the most interesting work I could possibly be doing right now. So, don't feel sorry for me. Really."`
+			branch surname
+				has "last name: Winters"
+			goto conclusion
```

Comment:
```suggestion
				has "last name: Winters"
			`	You talk for a while longer, and she tells you she'll be sure to be in touch if they ever get to the root of the Pug's motivation. "Until then," she says, "good luck in your travels. I hope the galaxy never runs out of ways to surprise you."`
				decline
```

Theoretically, using the goto should work just fine, and is preferable. But my mission-troubleshooting is rusty, and simply duplicating this line is the only way I can find to not generate an error and have the mission work the way it is supposed to.